STF 1.1 documentation updates (round 1) (#136)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_configuring-ephemeral-storage.adoc

[id='configuring-ephemeral-storage_{context}']
= Configuring ephemeral storage

> To configure {ProjectShort} components for ephemeral storage, add `...storage.strategy: ephemeral` to the corresponding parameter. For example, to enable ephemeral storage for the Prometheus backend, set `backends.metrics.prometheus.storage.strategy: ephemeral`. Components that support configuration of ephemeral storage include `alerting.alertmanager`, `backends.metrics.prometheus`, and `backends.events.elasticsearch`. You can add ephemeral storage configuration at installation time or, if you already deployed {ProjectShort}, complete the following steps:

Reviewer: ```suggestion
To configure {ProjectShort} components for ephemeral storage, add `storage: strategy: ephemeral` to the corresponding parameter. For example, to enable ephemeral storage for the Prometheus backend, set `backends: metrics: prometheus: storage: strategy:  ephemeral`. Components that support configuration of ephemeral storage include `alerting: alertmanager`, `backends: metrics: prometheus`, and `backends: events: elasticsearch`. You can add ephemeral storage configuration at installation time or, if you already deployed {ProjectShort}, complete the following steps:
```

+
[source,bash]
----
$ oc edit stf default
----

> . Add the `...storage.strategy: ephemeral` parameter to the `spec` section of the relevant component:

Reviewer: ```suggestion
. Edit the `spec` section of the relevant component and add `strategy: ephermeral` to the `alerting` and `backends` parameters:
```

> . Add the `...storage.strategy: ephemeral` parameter to the `spec` section of the relevant component:

Reviewer: ```suggestion
. Add the `storage: strategy: ephemeral` parameter to the `spec` section of the relevant component:
```
